@here/olp-sdk-core/lib/utils/BlobData.ts

```diff
@@ -38,7 +38,13 @@ export abstract class BlobData {
     /**
      * Gets the size of the data.
      *
-     * @returns The length of the data in bytes.
+     * @returns The `Promise` object with the length of the data in bytes.
      */
     abstract size(): number;
+
+    /**
+     * Should be called if exist at the end of the reading data for
```

Comment:
This is a very vague description.
```suggestion
     * This method should always be called at the end of the reading process
     * so that we can properly clean up.
```

Reply:
updated, tnx